test/validation/api/atomic/atomic.c

```diff
@@ -571,6 +689,7 @@ static void test_atomic_init(void)
 	odp_atomic_init_u64(&global_mem->a64u_max, 0);
 	odp_atomic_init_u32(&global_mem->a32u_xchg, 0);
 	odp_atomic_init_u64(&global_mem->a64u_xchg, 0);
+	test_atomic_init_u128(&global_mem->a128u, 0);
```

Comment:
`odp_atomic_store_u128()` is currently not tested.  The test should likely go into `test_atomic_store()`.

Reply:
Hi Matias, Thanks for the comments. Test case work is not complete yet. We have to modify the 128 bit test cases as per the latest change (init/load/store). Malvika will notify once it is complete.